modules/configuring-firewall.adoc

```diff
@@ -75,7 +75,10 @@ Before you install {product-title}, you must configure your firewall to grant ac
 |URL | Function
 
 |`mirror.openshift.com`
-|Required to access mirrored installation content and images
+|Required to access mirrored installation content and images.  Also a source of release image signatures, although the cluster-version operator needs only a single functioning source.
+
+|`storage.googleapis.com/openshift-release`
+|A source of release image signatures, although the cluster-version operator needs only a single functioning source.
```

Comment:
```suggestion
|Although the Cluster Version Operator requires only a single functioning content source, it also is a source of release image signatures.
```

Reply:
Let's sort out the wording in [this thread][1], and then come back to this entry.

[1]: https://github.com/openshift/openshift-docs/pull/19365#discussion_r374189442